addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2353,6 +2353,40 @@ QUnit.test('focusing a chat window of a chat should make new message separator d
     );
 });
 
+QUnit.test('livechat: closing a chat window with no message from admin side unpins it', async function (assert) {
```

Comment:
This test must be in `im_livechat` module where the feature is introduced

Reply:
ok for the copy in livechat but remove it from here

Reply:
Done

Reply:
The test is still here?

Reply:
That's weird, it's in `addons/im_livechat/static/src/components/chat_window_manager/chat_window_manager_tests.js` from what I know

Reply:
My bad, forgot to remove it when I copied the code in the correct file.